client/src/components/App.js

```diff
@@ -37,23 +38,38 @@ class App extends Component {
   state = {
     authenticated: false,
     currentUser: null,
+    firstName: null,
+    lastName: null,
+    currentEmail: null,
+    userUID: null,
     redirect: false
   };
 
   componentDidMount = () => {
     this.removeAuthListener = fire.onAuthStateChanged(user => {
       if (user) {
+        // Last # of occurrance of Space
+        let space = user.displayName.lastIndexOf(" "); 
```

Comment:
I think the only thing about splitting up a name by a single space is if someone has multiple spaces in their name: "Mary Ann Doe" or "Jean Claude Van Damme". Something we may have to consider and update in the future.

Reply:
Yes, but it would be much more prefered that their first name is still intact with Mary Ann and their last name is Doe. Otherwise, the split method doesn't work on the display name because it's already given to us as a split name. But this was the intention of this code was to have the whole name but split like so.